fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/WitnessDeleteListener.java

```diff
@@ -0,0 +1,6 @@
+package com.github.dedis.student20_pop.detail;
+
+public interface WitnessDeleteListener {
+
+  void onDeleteClicked(String witness);
```

Comment:
Wouldn't this be used for adding a witness instead of deleting?

Reply:
That's a separate event IIRC. This one's about handling witness deletion